src/states/common/bootstrapped.rs

```diff
@@ -99,12 +99,7 @@ pub trait Bootstrapped: Base {
     }
 
     fn resend_unacknowledged_timed_out_msgs(&mut self, token: u64) {
-        if let Some((unacked_msg, ack)) = self.ack_mgr_mut().find_timed_out(token) {
-            trace!("{:?} Timed out waiting for {:?}: {:?}",
-                   self,
-                   ack,
-                   unacked_msg);
-
```

Comment:
so far, we only interested in whether the msg got gave up.
and this trace does dump much duplicated logs.